splitio/sync/synchronizer.py

```diff
@@ -291,6 +299,8 @@ def sync_all(self):
             try:
                 if not self.synchronize_splits(None, False):
                     attempts -= 1
+                    how_long = self._backoff.get()
```

Comment:
if we are moving to backoff, there won't be attempts any longer. you can remove the attempts

Reply:
done